modules/rosa-autoscaling-nodes.adoc

```diff
@@ -0,0 +1,61 @@
+:_module-type: PROCEDURE
+//Specify the _module-type as either "CONCEPT, PROCEDURE, or REFERENCE"
+
+// Module included in the following assemblies:
+//
+// rosa-nodes/rosa-autoscale-nodes.adoc
+
+[id="rosa-autoscaling_{context}"]
+= Autoscaling nodes in a cluster
+
+[role="_abstract"]
+Configure autoscaling to increase the number of worker nodes that are available in a cluster. Autoscaling for worker nodes is configured in the machine pool definition.
+
+Autoscaling is dependent on the resource quotas that are set. Verify resource quotas and request quota increases from the link:https://aws.amazon.com/console/[AWS console].
+
+[NOTE]
+====
+Autoscaling can be disabled on a machine pool, but it cannot be disabled on a cluster. However, if no machine pool autoscaling is enabled, then autoscaling will not occur on that cluster even if autoscaling is enabled on the cluster.
+====
+
+.Procedure
+
+. To identify the machine pool IDs in a cluster, enter the following command:
++
+[source,terminal]
+----
+$ rosa list machinepools --cluster=<cluster_name>
+----
++
+.Example output
++
+[source,terminal]
+----
+ID        AUTOSCALING   REPLICAS    INSTANCE TYPE  LABELS    TINTS   AVAILABILITY ZONES
+mp1       No            2           m5.xlarge                        us-east-1a
+----
++
+Use the output to determine which machine pool ID to configure for autoscaling.
+
+. To enable autoscaling on a machine pool, using the default minimum and maximum of 2 replicas, enter the following command:
++
+[source,terminal]
+----
+$ rosa edit machinepool --cluster=<cluster_name> --name=<machinepool_ID> --enable-autoscaling
```

Comment:
This is not the correct syntax of the command. It should be `rosa edit machinepool --enable-autoscaling --min-replicas=<#min> --max-replicas=<#max> --cluster=<cluster-name> <machine-pool-name>`. Also you must give it a min and max for replicas.  It does not assign a "default".

Reply:
It used to have a default if I recall correctly, but that seems to have changed. I fixed it. Also, Victor told me that the order of the flags does not matter, but I have changed it to match yours anyway.

Reply:
OK, I checked with Victor and the defaults are only when creating a cluster and enabling autoscaling there. That was my point of confusion. Also, he said the order of the flags/aruguments in the syntax does not matter. If you're getting an error, it's probably a bug.